scipy/signal/fir_filter_design.py

```diff
@@ -708,23 +708,72 @@ def remez(numtaps, bands, desired, weight=None, Hz=None, type='bandpass',
 
     Examples
     --------
-    For a signal sampled at 100 Hz, we want to construct a filter with a
-    passband at 20-40 Hz, and stop bands at 0-10 Hz and 45-50 Hz. Note that
-    this means that the behavior in the frequency ranges between those bands
-    is unspecified and may overshoot.
+    In those examples the remez function gets used creating a bandpass-, bandstop-, low- and highpassfilter.
+    The used parameters are the filter order, an array with according frequency boundaries, the desired attenuation values and the sampling frequency.
+    Using signal.freqz the according frequency response gets calculated and plotted.
 
+    >>> import numpy as np
     >>> from scipy import signal
-    >>> fs = 100
-    >>> bpass = signal.remez(72, [0, 10, 20, 40, 45, 50], [0, 1, 0], fs=fs)
-    >>> freq, response = signal.freqz(bpass)
-
     >>> import matplotlib.pyplot as plt
-    >>> plt.semilogy(0.5*fs*freq/np.pi, np.abs(response), 'b-')
-    >>> plt.grid(alpha=0.25)
-    >>> plt.xlabel('Frequency (Hz)')
-    >>> plt.ylabel('Gain')
+
+    >>> def plot_response(fs, w, h, title):
+    ...     plt.figure()
+    ...     plt.ylim(-40, 5)
+    ...     plt.grid(True)
+    ...     plt.xlabel('Frequency (Hz)')
+    ...     plt.title(title)
```

Comment:
this function doesn't make much sense, nothing is plotted. did you lose some lines here?

Also, better use the object-oriented matplotlib API rather than `plt.` so:
```
fig = plt.figure()
ax = fig.add_subplot(111)
ax.plot(...etc)
```